lighthouse-core/computed/computed-artifact.js

```diff
@@ -11,7 +11,7 @@ const log = require('lighthouse-logger');
 /**
  * Decorate computableArtifact with a caching `request()` method which will
  * automatically call `computableArtifact.compute_()` under the hood.
- * @template {{name: string, compute_(artifacts: unknown, context: LH.Audit.Context): Promise<unknown>}} C
+ * @template {{name: string, compute_(artifacts: unknown, context: LH.Gatherer.ComputedContext): Promise<unknown>}} C
```

Comment:
I don't know that it belongs in either `Audit` or `Gatherer`, maybe in `Artifacts` namespace?

Reply:
Yeah `Artifacts` sounds better.